releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -1087,8 +1097,8 @@
   },
   "124091": {
     "commit": "bbfd2145dee670458d623800567facca13e27cc3",
-    "text": "kubelet no longer crashes when a DRA driver returns a nil as part of the Node(Un)PrepareResources response instead of an empty struct (did not affect drivers written in Go, first showed up with a driver written in Rust).",
-    "markdown": "Kubelet no longer crashes when a DRA driver returns a nil as part of the Node(Un)PrepareResources response instead of an empty struct (did not affect drivers written in Go, first showed up with a driver written in Rust). ([#124091](https://github.com/kubernetes/kubernetes/pull/124091), [@bitoku](https://github.com/bitoku)) [SIG Node]",
+    "text": "kubelet no longer crashes when a Dynamic Resource Allocation (DRA) driver returns a nil as part of the Node(Un)PrepareResources response instead of an empty struct (did not affect drivers written in Go, first showed up with a driver written in Rust).",
+    "markdown": "Kubelet no longer crashes when a Dynamic Resource Allocation (DRA) driver returns a nil as part of the Node(Un)PrepareResources response instead of an empty struct (did not affect drivers written in Go, first showed up with a driver written in Rust). ([#124091](https://github.com/kubernetes/kubernetes/pull/124091), [@bitoku](https://github.com/bitoku))",
```

Comment:
```suggestion
    "markdown": "Kubelet no longer crashes when a Dynamic Resource Allocation (DRA) driver returns a nil as part of the Node(Un)PrepareResources response instead of an empty struct (did not affect drivers written in Go, first showed up with a driver written in Rust). ([#124091](https://github.com/kubernetes/kubernetes/pull/124091), [@bitoku](https://github.com/bitoku)) [SIG Node]",
```